keps/sig-api-machinery/2876-crd-validation-expression-language/README.md

```diff
@@ -1064,7 +1065,8 @@ Validation expressions will be enforced again.
 
 ###### Are there any tests for feature enablement/disablement?
 
-These will be introduced in the Alpha implementation.
+Both unit test and integration test are introduced in the Alpha implementation.
+We will be adding e2e test.
```

Comment:
are we actually adding an e2e test with feature disablement? that's usually the trickiest place to test enablement/disablement since it requires modifying cluster config and restarting the API server

unit/integration tests with enablement/disablement is more what I expected

Reply:
We plan to add e2e test in beta phrase when the feature is enabled by default. I am not sure if I got the message right. Do you think the e2e test is optional for beta?

Reply:
an e2e test is good but that belongs more under the test plan section

this is specifically about testing the behavior on a `disabled → enabled` or `enabled → disabled` feature boundary, which I don't think e2e tests are great for

Reply:
We already have a integration test to verify enabled=false (https://github.com/kubernetes/kubernetes/blob/7477e1c91a141f33e1895427f4710f13cf7daaf8/test/integration/apiserver/crd_validation_expressions_test.go#L45). Maybe augment this with the (disabled->enabled, enabled->disabled) transition tests?

Reply:
Thanks for pointing out. Sorry for the confusion. I guess I updated in the wrong section. Reverted it back.